packages/react-components/src/components/SendBox.tsx

```diff
@@ -369,7 +374,7 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
       props.onRenderAttachmentUploads()
     ) : (
       <Stack className={attachmentUploadCardsStyles}>
-        <FluentV9ThemeProvider v8Theme={theme}>
+        <FluentV9ThemeProvider v8Theme={theme} className={customV9Styles.clearBackground}>
```

Comment:
It looks like /* @conditional-compile-remove(file-sharing-acs) */ is missed here